pkg/kube/vault/vault.go

```diff
@@ -57,6 +60,25 @@ type GCSConfig struct {
 	HaEnabled string `json:"ha_enabled"`
 }
 
+// AWSConfig keeps the vault configuration for AWS
+type AWSConfig struct {
```

Comment:
Please define the tags fro these fields.

Reply:
What tags? The equivalent type for Google Cloud (`GCPConfig`) doesn't have any tags for the fields.